### ISSUE TITLE
Nullish Coalescing Operator not transpiled in Parcel v2

### ISSUE THREAD

**Issue by RPDeshaies** [parcel-bundler/parcel#6114]
# 🐛 bug report

When using the Nullish Coalescing operator, Parcel v2 doesn't transpile the code so that it's supported on all browsers.

What I find interesting is that parcel uses babel's preset-env end this plugin is included inside present-env

https://v2.parceljs.org/languages/babel/#extending-the-default-babel-config

https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator

![image](https://user-images.githubusercontent.com/6224111/114312075-575bd800-9abf-11eb-97be-ee1329c96da0.png)

## 🎛 Configuration (.babelrc, package.json, cli command)

No specific babel RC config file

```js
    "start": "parcel serve ./lib/index.html",
    "build": "parcel build ./lib/index.html",
```

```js
    "@babel/core": "^7.13.15",
    "@parcel/transformer-image": "2.0.0-beta.2",
    "@parcel/transformer-typescript-tsc": "^2.0.0-alpha.3",
    "parcel": "^2.0.0-beta.2",
    "typescript": "^4.1.4"
```

## 🤔 Expected Behavior

Code using `??` should be transpiled to something like what is described in the babel doc

![image](https://user-images.githubusercontent.com/6224111/114312108-78242d80-9abf-11eb-87bd-6dee92a99143.png)

## 😯 Current Behavior

```
Uncaught SyntaxError: Unexpected token '??'
```

![image](https://user-images.githubusercontent.com/6224111/114312141-9b4edd00-9abf-11eb-8fae-af4fa9587001.png)
![image](https://user-images.githubusercontent.com/6224111/114312123-8a05d080-9abf-11eb-965e-c7a98d124f67.png)

## 💁 Possible Solution

Hard to suggest without first knowing where the bug comes from, but I would guess "fix parcel's config so that babel's preset env is properly used to be able to transpile the `??` operator in typescript.

## 🔦 Context

So using the `??` operator gives me the ability to quickly check for undefined values without having to worry about javascript"s `||` interpretation of what is "false"

## 💻 Code Sample

Have parcel load an html file which loads a typescript file which includes

```
const toto = "" ?? 3;
```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |  ^2.0.0-beta.2|
| Node             | v14.11.0|
| npm/Yarn         | 1.22.5|
| Operating System | MacOS Catalina 10.15.7|

**Comment by mischnic** [comment]
Have you set a browserslist config? Does it actually contain a browser that doesn't support nullish coalescing?

https://v2.parceljs.org/getting-started/webapp/#browserslist

**Comment by danieltroger** [comment]
Can confirm it transpiles for me so you must be having a configuration issue. Probably helps more to read the docs like @mischnic suggested but I don't want to be entirely unhelpful so if you want some example configurations check [a .zip file in one of the issues I opened](https://github.com/parcel-bundler/parcel/issues?q=is%3Aissue+author%3Adanieltroger+). If it's an issue about IE11 my repro code for sure is configured to transpile.

Edit: also, @RPDeshaies, it seems like you're new to parcel 2. Might I suggest making use of the [multiple targets](https://v2.parceljs.org/blog/alpha1/#multiple-targets%2C-and-bundle-splitting) feature? No need to hammer us 95% with a modern browser with transpilation and polyfills

**Comment by RPDeshaies** [comment]
@mischnic I'm using google chrome (which supports this feature) and had a browser list >0.2% but after some more investigation I found the problematic line of code inside a react component

```
       <Container maxWidth={("lg" || props.maxWidth) ?? "md"}>
```

Which isn't valid code at all since "lg" is always going to be truthy. 

Though I still find it interesting that parcel 1 transpiled it differently than parcel v2 in a way that was valid for the browser.

I'll fix the code on my end, sorry for bothering you all. Closing this issue and thanks for the help :)
